R/endpoints.R

```diff
@@ -17,13 +17,13 @@ api_build <- function(queue) {
   pr$handle("GET", "/meta/plotting/<iso3>", endpoint_plotting_metadata,
             serializer = serializer_json_hintr())
   pr$handle("GET", "/download/spectrum/<id>", endpoint_download_spectrum(queue),
-            serializer = serializer_zip())
```

Comment:
should we always be returning the same filename here? or should it depend on the id? with the same filename in the odin ui, users ended up confused with an ever-growing number of filenames ending in `(1)`, `(2)` etc

Reply:
Hmm I'm not sure what is good practice here. I've never thought having filenames with a `(1)` and `(2)` was that confusing but I guess it might be. Let's ask the others in slack.